src/logger.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as vscode from "vscode";
+import TelemetryReporter from "vscode-extension-telemetry";
+
+export enum Type {
+    EXCEPTION = "exception",
+    USAGEDATA = "usageData",
+    USAGEERROR = "usageError",
+    ACTIVATEEXTENSION = "activateExtension",

```

Comment:
This is very specific and stands out from the other enum members. I think activation belongs to usage data.

Reply:
Add the activation event to usage data. And keep the old activation event for temporary transition.